react-virtualized/index.d.ts

```diff
@@ -120,7 +120,7 @@ declare module "react-virtualized" {
         rowCount: number;
         rowGetter?: (info: { index: number }) => any;
         rowHeight: number | ((info: { index: number }) => number);
-        rowRenderer?: (info: { index: number, isScrolling: boolean }) => React.ReactNode;
+        rowRenderer?: (info: { index: number, isScrolling?: boolean, key: any, style: string }) => React.ReactNode;
```

Comment:
I actually made the same change so I can just delete my PR, but I'm not 100% sure key exists on Table rowRenderer, just based on the docs. https://github.com/bvaughn/react-virtualized/blob/master/docs/Table.md